src/content/docs/it/guides/deploy/cloudflare.mdx

```diff
@@ -0,0 +1,138 @@
+"preview": "wrangler pages dev ./dist"
+```
+
+## Come effettuare il deploy di un sito SSR
+
+Puoi creare un sito Astro SSR e eseguire il deploy su Cloudflare Pages utilizzando l'adattatore [`@astrojs/cloudflare`](/it/guides/integrations-guide/cloudflare/).
+
+Segui i passaggi qui sotto per configurare l'adattatore. Successivamente, puoi effettuare il deploy del tuo sito utilizzando uno dei metodi visti in precedenza.
+
+### Installazione rapida
+
+Aggiungi l'adattatore di Cloudflare per abilitare l'SSR nel tuo progetto Astro utilizzando il seguente comando: `astro add`. Questo installerà l'adattatore e apporterà le modifiche appropriate al tuo file `astro.config.mjs` in un solo passaggio.
+
+```bash
+npx astro add cloudflare
+```
+
+### Installazione manuale
+
+Se preferisci installare manualmente l'adattatore, segui questi due passaggi:
+
+1. Aggiungi l'adattatore `@astrojs/cloudflare` alle dipendenze del tuo progetto utilizzando il tuo package manager preferito. Se stai utilizzando npm o non sei sicuro di quale tu stia utilizzando, esegui questo comando nel terminale:
+
+    ```bash
+    npm install @astrojs/cloudflare
+    ```
+
+2. Aggiungi questi parametri al tuo `astro.config.mjs`:
+
+    ```js title="astro.config.mjs" ins={2, 5-6}
+    import { defineConfig } from 'astro/config';
+    import cloudflare from '@astrojs/cloudflare';
+
+    export default defineConfig({
+      output: 'server',
+      adapter: cloudflare()
+    });
+    ```
+
+### Modalità
+
+Attualmente sono supportate due modalità quando si utilizzano le Funzioni di Cloudflare Pages con l'adattatore [`@astrojs/cloudflare`](https://github.com/withastro/astro/tree/main/packages/integrations/cloudflare#readme).
+
+1. **Modalità avanzata**: Questa modalità viene utilizzata quando si vuole eseguire una funzione in modalità `advanced`, che utilizza il file `_worker.js` in `dist`, o in modalità directory in cui le pagine compileranno il worker da una cartella `functions` nella cartella principale del progetto.
+
+   Se nessuna modalità viene impostata, il valore predefinito è `"advanced"`.
+
+2. **Modalità directory**: Questa modalità viene utilizzata quando si desidera eseguire una funzione in modalità `directory`, il che significa che l'adattatore compilerà la parte lato client della tua app nello stesso modo, ma sposterà lo script del worker in una cartella `functions` nella cartella principale del progetto. L'adattatore metterà sempre un file `[[path]].js` in quella cartella, consentendoti di aggiungere plugin e middleware aggiuntivi per le pagine.
+
+   ```ts title="astro.config.mjs" "directory"
+   export default defineConfig({
+     adapter: cloudflare({ mode: "directory" }),
+   });
+   ```
+
+### Utilizzo delle Funzioni di Cloudflare Pages
+
+Le [Funzioni di Pages](https://developers.cloudflare.com/pages/platform/functions/) ti consentono di eseguire codice lato server per abilitare funzionalità dinamiche senza dover usare un server dedicato.
```

Comment:
I would write `funzioni` not capitalized. There's an [interesting comment in the glossary PR](https://github.com/withastro/docs/pull/4201/files#diff-dfc8609ed04fccb77e428c6a450c820fdebbf9ed6dcba6dd0b51d0a77ef49979R18) - which is not merged yet - that we should capitalize names of products or technologies/languages .
In this specific example `Pages` - which is the name of the product - should be capitalized, while other more generic terms - like `functions` should be left uncapitalized.

Reply:
The product name is "Pages Functions", though. I am not sure if it should be uncapitalised.

https://developers.cloudflare.com/pages/platform/functions/